IFC4x3/Properties/e/ElectroMagneticStandardsCompliance_2waDzwzkPEmfBnDmVCqDVK/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="ElectroMagneticStandardsCompliance_2waDzwzkPEmfBnDmVCqDVK" Name="ElectroMagneticStandardsCompliance" UniqueId="ba90df7a-f6e6-4ec2-92f1-3707ccd0d7d4" PrimaryDataType="IfcLabel" />
```

Comment:
The compliance properties all have `IfcLabel` isnt this character restricted. is it worth having a longer field or coverting these to external references?